yo1k/tic_tac_toe/core/util.py

```diff
@@ -1,6 +1,12 @@
-def eq(cls):
+from __future__ import annotations
+from typing import TypeVar, Type
+
+T = TypeVar("T")  # pylint: disable=C0103
```

Comment:
`Pylint 2.12` (or earlier versions) [does not work properly](https://github.com/PyCQA/pylint/issues/3401) with `TypeVar` in the  (it is fixed in the 2.13).

Reply:
Could you please create a GitHub issue to update to Pylint 2.13 when it comes out, and to remove the suppression here?